starter/script.js

```diff
@@ -1 +1,413 @@
+					scene.rightDoor = scene.rightDoor.toUpperCase();
+				}
+				while (scene.rightDoor !== "R" && scene.rightDoor !== "C" && scene.rightDoor !== "D");
+				encounter();
+				break;
+			case "R":
+				reset();
+				scene.firstScene = true;
+				break;
+			default:
+				break;
+		}
+
+			// Go Right 
+
+		switch(scene.rightPassage) {
+			case "W":
+				reset();
+				do {
+					scene.waiting = prompt("Pulling out your gun, you slowly make your way in. There is a TV switched on. Do you\n(S)witch off the TV;\n(H)ide and wait to see if anyone else is coming; or\n(G)o back outside the room:");
+					scene.waiting = scene.waiting.toUpperCase();
+				}
+				while (scene.waiting !== "S" && scene.waiting !== "H" && scene.waiting !== "G");
+				encounter();
+				break;
+			case "G":
+				reset();
+				do {
+					scene.hostageRoom = prompt("It is locked. There is a number code you need to punch in. What do you enter: (Enter 'Q' to stop)");
+				}
+				while (scene.hostageRoom !== "Q" && scene.hostageRoom !== "q" && scene.hostageRoom !== passCode);
+				switch (true) {
+					case (scene.hostageRoom === "Q" || scene.hostageRoom === "q"):
+						reset();
+						scene.buildingAtrium = "G";
+						break;
+					case (scene.hostageRoom === passCode):
+						scene.breakCode = true;
+						break;
+					default:
+						break;
+				}
+				encounter();
+				break;
+			case "S":
+				reset();
+				do {
+					scene.shootToKill = prompt("Just when you pull out your gun, a soldier appear and you shoot him in the head. Phew! Do you\n(S)earch his body;\n(R)un along; or\n(T)his is only your imagination and you have not shot anyone:");
+					scene.shootToKill = scene.shootToKill.toUpperCase();
+				}
+				while (scene.shootToKill !== "S" && scene.shootToKill !== "R" && scene.shootToKill !== "T");
+				encounter();
+				break;
+			case "R":
+				reset();
+				scene.firstScene = true;
+				break;
+			default:
+				break;
+		}
+
+
+		
+
+
+		// Third Scene
+
+				// Take the Stairs
+
+		switch (scene.upstairs) {
+			case "T":
+				alert("It is stuck. You quickly use a knife to scrape it off. You notice there is a dose of adrenaline at your feet. You pick it up and shoot yourself with it. Time to head back.");
+				hitpoints = 200;
+				alert("Bonus: full hitpoints!");
+				showClue(0);
+				encounter();
+				reset();
+				scene.roomOnLeft = "T";
+				break;
+			case "F":
+				alert("You come to a dead end with a table and a cabinet. The cabinet is empty, however you notice there is a dog tag on the table. It reads '2LT JASON SMITH'. You pray for his safety as you turn back.");
+				showClue(1);
+				encounter();
+				reset();
+				scene.roomOnLeft = "T";
+				break;
+			case "G":
+				reset();
+				scene.buildingAtrium = "O";
+				break;
+			default:
+				break;
+		}
+
+				// Go Through the Door
+
+		switch (scene.rightDoor) {
```

Comment:
I think you asked this during the morning stand up.

When you do "break;"   - you break out of the most immediate {  code block  }
to break the while loop you need to set your variable gameContinue = false;